### PR TITLE
Documentation for Alexa demo with hands-free support

### PR DESCRIPTION
**Pull request by hpsaturn**

### CODE REVIEW

File: docs/Alexa/MatrixCreator.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+## Let's get started
+The original Alexa on a Pi project required manual download of libraries/dependencies and updating a series of configuration files that were prone to human error. To make the process faster and easier, we’ve included an install script with the project that will take care of all the heavy lifting. Not only does this reduce setup time to less than an hour on a Raspberry Pi 3, it only requires developers to adjust three variables in a single install script -
+
+### Step 1: Setting up your Pi
+Configure your RaspberryPi like a original Alexa documentation, for this please completing steps: **1,2,3,4,5 and 6** from original documentation: [Raspberry Pi][2]
+
+---
+
+### Step 2: Override ALSA configuration
+On your RaspberryPi home: `/home/pi` edit `.asoundrc` file and put this (If your prefer, make a backup):
```
Comment (kdpatino) [inline docs/Alexa/MatrixCreator.md]: Maybe mention that matrix creator has 8 individual channels and one more with delay and sum algorithm.

File: docs/Alexa/MatrixCreator.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+4. **Micro SD Card** (Minimum 8 GB) - You need an operating system to get started. NOOBS (New Out of the Box Software) is an easy-to-use operating system install manager for Raspberry Pi. The simplest way to get NOOBS is to buy an SD card with NOOBS pre-installed - [Raspberry Pi 8GB Preloaded (NOOBS) Micro SD Card](https://www.amazon.com/gp/product/B00ENPQ1GK/ref=oh_aui_detailpage_o01_s00?ie=UTF8&psc=1). Alternatively, you can download and install it on your SD card.
+5. **External Speaker** with 3.5mm audio cable - [Buy on Amazon](http://amzn.com/B007OYAVLI)
+6. A **USB Keyboard & Mouse**, and an external **HDMI Monitor** - we also recommend having a USB keyboard and mouse as well as an HDMI monitor handy if you're unable to [remote(SSH)][1] into your Pi.
+7. Internet connection (Ethernet or WiFi)
+8. (Optional) WiFi Wireless Adapter for Pi 2 ([Buy on Amazon](http://www.amazon.com/CanaKit-Raspberry-Wireless-Adapter-Dongle/dp/B00GFAN498/)).
+   **Note:** Pi 3 has built-in WiFi.
+
+For extra credit, we'll show you how to [remote(SSH)][1] into your device, eliminating the need for a monitor, keyboard and mouse - and how to tail logs for troubleshooting.
```
Comment (eighteyes) [inline docs/Alexa/MatrixCreator.md]: This doesn't happen.

File: docs/Alexa/MatrixCreator.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+![](/docs/Alexa/matrix-creator-alexa.png)
+
+This guide provides step-by-step instructions for setting up AVS on a **Raspberry Pi** with a [MATRIX Creator](https://creator.matrix.one/) hat. It demonstrates how to access and test AVS using our Java sample app (running on a Raspberry Pi), a Node.js server, and a third-party wake word engine using MATRIX mic array. You will use the Node.js server to obtain a Login with Amazon (LWA) authorization code by visiting a website using your Raspberry Pi's web browser.
+
+## Required hardware
+
+Before you get started, let's review what you'll need.
+
+1. **Raspberry Pi 3** (Recommended) or **Pi 2 Model B** (Supported)  - Buy at Amazon - [Pi 3](https://amzn.com/B01CD5VC92) or [Pi 2](http://amzn.com/B00T2U7R7I).
+2. **MATRIX Creator** - Raspberry Pi does not have a built-in microphone, with MATRIX Creator mic array your obtain 8 mic for Alexa - [Buy MATRIX Creator](https://creator.matrix.one/#!/buy)
```
Comment (eighteyes) [inline docs/Alexa/MatrixCreator.md]: replace last sentence fragment

, the MATRIX Creator has an 8 mic array

File: docs/Alexa/MatrixCreator.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+3. **Micro-USB power cable** for Raspberry Pi.
+4. **Micro SD Card** (Minimum 8 GB) - You need an operating system to get started. NOOBS (New Out of the Box Software) is an easy-to-use operating system install manager for Raspberry Pi. The simplest way to get NOOBS is to buy an SD card with NOOBS pre-installed - [Raspberry Pi 8GB Preloaded (NOOBS) Micro SD Card](https://www.amazon.com/gp/product/B00ENPQ1GK/ref=oh_aui_detailpage_o01_s00?ie=UTF8&psc=1). Alternatively, you can download and install it on your SD card.
```
Comment (eighteyes) [inline docs/Alexa/MatrixCreator.md]: Should we mention that Raspbian is also supported?

File: docs/Alexa/MatrixCreator.md (hunk 2 above)
Comment (hpsaturn) [inline docs/Alexa/MatrixCreator.md]: mmm but for example I can used this because monitor and keyboard are extra devices and are not necessaries

Comment (eighteyes) [inline docs/Alexa/MatrixCreator.md]: Oh, I see what happened here. Replace 'we'll show you how to' with 'enable'

File: docs/Alexa/MatrixCreator.md (hunk 4 above)
Comment (hpsaturn) [inline docs/Alexa/MatrixCreator.md]: NOOBs installer get Raspbian.

File: docs/Alexa/MatrixCreator.md (hunk 1 above)
Comment (hpsaturn) [inline docs/Alexa/MatrixCreator.md]: ready

**Pull request merged**